core/bin/snapshots_creator/src/main.rs

```diff
@@ -19,6 +18,7 @@ use zksync_config::{
 use zksync_dal::{ConnectionPool, Core};
 use zksync_env_config::{object_store::SnapshotsObjectStoreConfig, FromEnv};
 use zksync_object_store::ObjectStoreFactory;
+use zksync_prometheus_exporter::PrometheusExporterConfig;
```

Comment:
Future work (?): I wonder whether it makes sense to fold Prometheus exporter into `zksync_vlog`; seems that they are almost always used together.